lib/src/connector/connection.cc

```diff
@@ -258,7 +258,7 @@ void Connection::send(const std::string& msg)
     websocketpp::lib::error_code ec;
     endpoint_->send(connection_handle_,
                     msg,
-                    websocketpp::frame::opcode::binary,
+                    websocketpp::frame::opcode::text,
```

Comment:
Preliminary results show that v2 might be faster than v1 in the broker, although I'm not sure how much I trust Gatling on this yet. I've created [PCP-683](https://tickets.puppetlabs.com/browse/PCP-683) to follow up on it.